src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -731,6 +732,25 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
     });
   }
 
+  /**
+   * Uses the CK Editor (ready) to link a file upload handler to the
+   * CK Editor instance.
+   * 
+   * @param eventData Object type added and used by CK Editor
+   * @returns {Promise}
+   */
+  public editorOnReady(eventData) {
+    // We need to grab our vars explicitely and pass them through to the CK Editor function.
+    const projectId = this.projectId;
+    const documentService = this.documentService;
+    const pathAPI = this.pathAPI;
+    eventData.plugins.get('FileRepository').createUploadAdapter = function (loader) {
```

Comment:
```suggestion
    eventData.plugins.get('FileRepository').createUploadAdapter = (loader) => {
```

Not a big deal for now, but part of the work of updating this app is using modern coding standards and being syntactically consistent. Consider using arrow functions here.

Reply:
Interesting, this was a copy paste from CKEditor, but I notice that they use (loader) => as well. Not sure where I found the older version from :)